src/common/python/constants.py

```diff
@@ -7,12 +7,14 @@
 TEMP_POSTFIX = "-temp"
 ERROR = "Error"
 CAUSE = "Cause"
+RDS_SNAPSHOTS_BUCKET_NAME_PREFIX = "rds-snapshots-"
 
 DELETE = "Delete"
 RENAME = "Rename"
 SNAPSHOT = "SnapshotCreation"
 DB_RESTORE = "Restore"
 CLUSTER_RESTORE = "ClusterRestore"
+EXPORT_SNAPSHOT = "SnapshotExportTask"
```

Comment:
Can we remove the Task from SnapShotExportTask as the name doesn't follow the convention.

Reply:
yes, will do

Reply:
done, 
https://github.com/intuit/Trapheus/pull/5/commits/4d228832296da832e7947003027768c025dbafb0